routes/password_check.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
```

Comment:
@chilts and @rfk - this is a new module to check passwords, take care of notifying customs, and lock the account if necessary. Seem reasonable?